index.md

```diff
@@ -39,6 +40,10 @@ There are several FAIR Data Point implementations in existence:
 * [RD-Nexus](https://rd.discovery-nexus.org) implements the protocol.
 * [LOVD](https://www.lovd.nl) implements the protocol.
 
+There are also different implementations of the metadata harvesting protocol:
+* The [reference implementation](https://github.com/FAIRDataTeam/FAIRDataPoint) as described above.
```

Comment:
I'm not sure what the intended message is here. Do we mean to refer to the index functionality here?